bluepysnap/morph.py

```diff
@@ -55,23 +57,39 @@ def __init__(self, morph_dir, nodes):
             self._load = lru_cache(maxsize=MORPH_CACHE_SIZE)(self._load)
 
     def _is_biophysical(self, node_id):
-        return self._nodes.get(node_id, Node.MODEL_TYPE) == "biophysical"
+        return self._population.get(node_id, Node.MODEL_TYPE) == "biophysical"
 
     def get_filepath(self, node_id):
-        """Return path to SWC morphology file corresponding to `node_id`."""
-        name = self._nodes.get(node_id, Node.MORPHOLOGY)
+        """Return path to SWC morphology file corresponding to `node_id`.
+
+        Args:
+            node_id (int/CircuitNodeIds): could be a single int or a CircuitNodeIds of length 1.
```

Comment:
I really don't like it. int and CircuitNodeIds are not homogeneous. One is a scalar and the other one is homogeneous to a sequence. 

I would prefer having an [] accessor in the CircuitNodeIds class and having support of the following:
```python
morph_helper.get_filepath(circuit_node_ids[0])
```
This support would also enable this:
```python
filepaths = [morph_helper.filepath(node_id) for node_id in circuit_node_ids]
```

Reply:
I changed this for CircuitNodeId both are homogeneous to a single id